.travis.yml

```diff
@@ -1,9 +1,9 @@
-
 language: php
 
 php:
-  - '7.0'
-  - '7.2.0RC5'
+  - 7.0
```

Comment:
Please keep using strings. `7.0` == `7` == `7.x.x`, `'7.0'` == `7.0.x`

Reply:
What? Didn't know about that. Do you know where I can find this in Travis documentation?

Reply:
I don't think that is a Travis thing. In YAML if you don't quote `7.0` it's a floating point number so it is the same as `7`. Now whatever version manager Travis uses to install runtimes will get `7`, not `7.0`, so specifying `7.0` is at least misleading. I know that `nvm install 7` will install the latest `7.x.x` version, while `nvm install 7.0` will install the latest `7.0.x` version. I don't know how it is for PHP but there must have been a reason for this commit https://github.com/felixfbecker/php-language-server/commit/41e84880b380368ae5862c01e9a00d55067c48ec and there is no disadvantage to using strings.

Reply:
Oh, I had no idea. Thanks for taking the time to explain.